src/cascadia/TerminalApp/LeafPane.idl

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import "IPane.idl";
+
+namespace TerminalApp
+{
+    enum Borders2
+    {
+        None = 0,
+        Top = 1,
+        Bottom = 2,
+        Left = 4,
+        Right = 8
+    };
```

Comment:
Why `Borders2` instead of `Borders`? (sorry if this wasn't a change you did)

Reply:
Fixed! Thanks for catching, was a relic of the initial implementation (to avoid collision with the one defined in `pane.h`